coordinator/src/services/translation/request/return/return.ts

```diff
@@ -14,7 +13,8 @@ export async function convertTaskToDispenseProposalReturn(
   task: fhir.Task,
   logger: pino.Logger
 ): Promise<hl7V3.DispenseProposalReturn> {
-  const id = createIdFromTaskIdentifier(task.identifier)
+  const idValue = getMessageId(task.identifier)
```

Comment:
I think we'll need to pass in the FHIR path if we do this. Otherwise any error messages will refer to `Bundle.identifier`